pool/pool.py

```diff
@@ -79,11 +80,12 @@ def __init__(
         self.info_logo_url = pool_config["pool_info"]["logo_url"]
         self.info_description = pool_config["pool_info"]["description"]
         self.welcome_message = pool_config["welcome_message"]
+        self.config_store = pool_config["store"] or SqlitePoolStore()
```

Comment:
That wont work, you probably need to do something like globals()[pool_config["store"]]

Reply:
Fixed it. code all working in my dev env